DSCResources/MSFT_xWebsite/MSFT_xWebsite.psm1

```diff
@@ -39,6 +49,11 @@ VerboseTestTargetFalseEnabledProtocols = Enabled Protocols for website "{0}" do
 VerboseTestTargetFalseDefaultPage = Default Page for website "{0}" does not match the desired state.
 VerboseTestTargetTrueResult = The target resource is already in the desired state. No action is required.
 VerboseTestTargetFalseResult = The target resource is not in the desired state.
+VerboseTestTargetFalsePreload = Preload for website "{0}" do not match the desired state.
+VerboseTestTargetFalseAutoStart = AutoStart for website "{0}" do not match the desired state.
+VerboseTestTargetFalseAutoStartProvider = AutoStartProvider for website "{0}" does not match the desired state.
+VerboseTestTargetFalseSSLFlags = SSLFlags are not in the desired state.
```

Comment:
xSSLSettings sets the SSL Settings for the website.

![image](https://cloud.githubusercontent.com/assets/651851/15080600/dc51086a-1373-11e6-83cf-51b12d18b710.png)

What SSL Flags changes is in the WebSite Bindings. Server Name Indication is the checkbox, SSLFlags is what its called in PowerShell

![image](https://cloud.githubusercontent.com/assets/651851/15080637/08d8ab5e-1374-11e6-9914-57b65ee0e7ff.png)


Could it be merged together? Probably. I created them seperately as that's how I had made it originally. Though xWebsite is begining to get quite large and may need to be broken up anyway.